File: codes/ALGraph/Python/ALGraphMain.py
import ALGraph as ag

graph = ag.ALGraph(7)

graph.AddEdge(ag.Vertex.A, ag.Vertex.B)
graph.AddEdge(ag.Vertex.A, ag.Vertex.D)
graph.AddEdge(ag.Vertex.B, ag.Vertex.C)
graph.AddEdge(ag.Vertex.D, ag.Vertex.C)
graph.AddEdge(ag.Vertex.D, ag.Vertex.E)
graph.AddEdge(ag.Vertex.E, ag.Vertex.F)
graph.AddEdge(ag.Vertex.E, ag.Vertex.G)

graph.ShowGraphEdgeInfo()

graph.DFShowGraphVertex(ag.Vertex.A)
print()
graph.DFShowGraphVertex(ag.Vertex.C)
print()
graph.DFShowGraphVertex(ag.Vertex.E)
print()
graph.DFShowGraphVertex(ag.Vertex.G)
print()

File: codes/ALGraph/Java/ALGraph/ALGraphMain.java
package ALGraph;

public class ALGraphMain {
    public static void main(String[] args) {
        ALGraph graph = new ALGraph(7);

        graph.AddEdge(Vertex.A, Vertex.B);
        graph.AddEdge(Vertex.A, Vertex.D);
        graph.AddEdge(Vertex.B, Vertex.C);
        graph.AddEdge(Vertex.D, Vertex.C);
        graph.AddEdge(Vertex.D, Vertex.E);
        graph.AddEdge(Vertex.E, Vertex.F);
        graph.AddEdge(Vertex.E, Vertex.G);

        graph.ShowGraphEdgeInfo(); System.out.println();

        System.out.println("DFS 결과");
        graph.DFShowGraphVertex(Vertex.A); System.out.println();
        graph.DFShowGraphVertex(Vertex.C); System.out.println();
        graph.DFShowGraphVertex(Vertex.E); System.out.println();
        graph.DFShowGraphVertex(Vertex.G); System.out.println();
        System.out.println();

        System.out.println("BFS 결과");
        graph.BFShowGraphVertex(Vertex.A); System.out.println();
        graph.BFShowGraphVertex(Vertex.C); System.out.println();
        graph.BFShowGraphVertex(Vertex.E); System.out.println();
        graph.BFShowGraphVertex(Vertex.G); System.out.println();
        System.out.println();
    }
}
